### PR TITLE
Supplement to the Russian translation + theme

### PR DESCRIPTION
**Pull request by giwih**
also added a black-amoled theme
photo_2024-11-02_23-15-26

### CODE REVIEW

**Comment by giwih** [comment]
also added a black-amoled theme 
photo_2024-11-02_23-15-26

**Comment by itz-rj-here** [comment]
image

The theme you added is not exeptable. Cause the parts of the weather icon and clock things, are on white color. The white icons too much effects the eyes too much. Which makes the theme bad. Also the theme doesn't goes with the text colors too. Sorry but I couldn't accept your new amoled theme.

**Comment by itz-rj-here** [comment]
> image
> 
> The theme you added is not exeptable. Cause the parts of the weather icon and clock things, are on white color. The white things too much effects the eyes too much. Which makes the theme bad. Also the theme doesn't goes with the text colors too. Sorry but I couldn't accept your new amoled theme.

**Comment by itz-rj-here** [comment]
@prem-k-r what you think about it?

**Comment by prem-k-r** [comment]
It's a nice idea, but need some modifications, I think. The background should be pure black, and other colors should be adjusted to complement the black. Additionally, need to review the shadow effects on the close button and shortcut icons to ensure they look good against the theme.

**Comment by itz-rj-here** [comment]
> It's a nice idea, but need some modifications, I think. The background should be pure black, and other colors should be adjusted to complement the black. Additionally, need to review the shadow effects on the close button and shortcut icons to ensure they look good against the theme.

Agreed.

**Review by itz-rj-here — CHANGES_REQUESTED** [review]
Modification needed.

**Comment by giwih** [comment]
{4F0BBD0A-6481-4C94-8AA1-3F454E9357CF}

**Comment by itz-rj-here** [comment]
image
Thanks but the shortcuts and the weather info section doesn't goes with it in all way. But still i need @prem-k-r confirmation what he thinks about it.

**Comment by giwih** [comment]
> shortcuts and the weather

how about this?
{2487E572-3CF4-4C61-A90B-DD288DC67526}

**Comment by itz-rj-here** [comment]
Great work. Now it looks good as new. Thanks for your hard work.

**Review by itz-rj-here — APPROVED** [review]

**Review by prem-k-r — CHANGES_REQUESTED** [review]
.colorsContainer input[value="dark"] {
    background-color: #818181;
}

line 1613, please change color to #434343

**Comment by prem-k-r** [comment]
How about this one? @giwih @itz-rj-here 

image

/* Amoled Dark */
    --bg-color-amoled: #000; 
    --accentLightTint-amoled: #434343;
    --darkerColor-amoled: #000000;
    --darkColor-amoled: #707070;
    --textColorDark-amoled: #969696;

**Comment by prem-k-r** [comment]
or a bit darker and maybe better
--accentLightTint-amoled: #353535;

**Comment by giwih** [comment]
> How about this one? @giwih itz-rj-here

also looks good, but personally I like my version a bit more) let itz-rj-here decide

**Comment by prem-k-r** [comment]
A bit darker than my last one

image

/* Amoled Dark */
    --bg-color-amoled: #000000; 
    --accentLightTint-amoled: #313131;
    --darkerColor-amoled: #000000;
    --darkColor-amoled: #696969;
    --textColorDark-amoled: #969696;

**Comment by prem-k-r** [comment]
image
image
image

/* Amoled Dark */
    --bg-color-amoled: #000000; 
    --accentLightTint-amoled: #1D1D1D;
    --darkerColor-amoled: #999999;
    --darkColor-amoled: #555555;
    --textColorDark-amoled: #aaaaaa;

**Comment by itz-rj-here** [comment]
> > How about this one? @giwih itz-rj-here
> 
> also looks good, but personally I like my version a bit more) let itz-rj-here decide

I will check it after i come to home.

**Comment by XengShi** [comment]
Concept for dark mode

**Comment by itz-rj-here** [comment]
image
Change this Weather temperature number color to full black, and change the shortcut to gray color like @prem-k-r just showed. and in other hand, In the menu section the github and feedback option of the text is hardly be seen. If possible please fix that issue too. 
image

**Comment by itz-rj-here** [comment]
I will come again after a few hours to check your commits/ changes. I need a nap. I just slept for only 4 hours last night cause of the exams presure. But luckily i got all my answers right thats all i need. 
I am going now. Bye.

**Comment by itz-rj-here** [comment]
> {2E4DA39A-4E90-4879-8A27-E65CB1601111} {E4CFF0CB-FC5C-4BA9-B43E-C2F6C3835C59}

Keep it open. I need to check more things. The amoled theme changes the shortcuts too. Bruh thats should have a different color theme selection. uhh.

**Review by itz-rj-here — CHANGES_REQUESTED** [review]
Needed an option to change only the shortcut themes not the all themes. Needed time to add the color theme on shortcut in different option.

**Comment by itz-rj-here** [comment]
@giwih The changes you made doesn't goes with the name of amoled theme. It completely changes the color theme. I would like to get some time to hopefully able to add a different option to add color code for shortcuts.

**Comment by prem-k-r** [comment]
Personally, I don't like black color in dark grey background as in clock and temperature.
I would suggest to go live with something like below for now
image
image
Considered shortcuts, toggles, texts, Github/feedback, etc

To the user, Amoled name is not showing anywhere. So user may think it as Dark grey, and existing dark theme to Light grey.
Later we can change some elements, have to creates separates color variables for seperate entities, and go for full amoled

well, @itz-rj-here, upto you guys

**Comment by giwih** [comment]
I like the gray temperature better too. But light gray shorcuts look not very good, as itz-rj-here said I added to be able to select color separately for shorcuts.
{3A5F85A0-5E73-4322-9ECD-952FB7E45EEC}

**Comment by itz-rj-here** [comment]
> I like the gray temperature better too. But light gray shorcuts look not very good, as itz-rj-here said I added to be able to select color separately for shorcuts. {3A5F85A0-5E73-4322-9ECD-952FB7E45EEC}
> 
> But I can't understand why there's a border, so if I add something lighter, it really stands out against the black background. {B8ACBAC9-7EBD-45C1-8A43-272F9502BEE0}

I know but we don't have any other options for now.

**Comment by prem-k-r** [comment]
@giwih 
image

**Comment by itz-rj-here** [comment]
image
Great work with the changes but fix the custom shortcut colors too. Then i will tell XengShi to merge this pr.

**Comment by prem-k-r** [comment]
> @giwih image

ah sorry, I missed to copy some changes in script.js. It's working normal.
image

It's great, thanks for your efforts.

icon color a little bit light grey will be better, I think
--shortcutIconColor-amoled:#5d5d5d;
image

**Comment by prem-k-r** [comment]
or how about this?
image
--shortcutColor-amoled: #414141;
--shortcutIconColor-amoled:#000;
image

**Comment by prem-k-r** [comment]
Also close button doesnt look good to me
maybe change back line 1744 to background-color: #fff
image
to (what it was earlier):
image

and if we can do something about the outer circle in close button 
image

**Comment by itz-rj-here** [comment]
@giwih can you resolve the conflicts please

**Review by itz-rj-here — APPROVED** [review]
Accepted @XengShi Can you please merge this pr.

File: script.js
[the remark this answers is not in the document]
Comment (giwih) [reply script.js:1]: what do you mean?

[the remark this answers is not in the document]
Comment (itz-rj-here) [reply script.js:1]: Well it was my mistake. I was watching your changes from mobile. Not from computer thats why i didn't saw some of the codes clearly.

**Review by prem-k-r — APPROVED** [review]

**Comment by itz-rj-here** [comment]
Bruh what happened again -_- why it again shows conflicts.

**Comment by itz-rj-here** [comment]
@giwih Can you resolve the conflicts one more time please
and
@XengShi Can you merge this pr please.

**Comment by prem-k-r** [comment]
Conflict resolved, good to go, @XengShi please merge

**Comment by itz-rj-here** [comment]
> Conflict resolved, good to go, @XengShi please merge

he is offline everywhere.

**Comment by XengShi** [comment]
Kindly make texts more readable/brighter, take some inspiration from the image

**Comment by itz-rj-here** [comment]
Hmm looks like i have to do something with this. 😐

**Comment by prem-k-r** [comment]
> Kindly make texts more readable/brighter, take some inspiration from the image

image
@XengShi How about this?

**Comment by itz-rj-here** [comment]
> > Kindly make texts more readable/brighter, take some inspiration from the image
> 
> image @XengShi How about this?

Yup as he wanted.

**Comment by itz-rj-here** [comment]
Currently it looks like this. 
image
image

**Comment by itz-rj-here** [comment]
Lets do a thing. @XengShi will merge this changes and you will fix the text color code.
What do you think? @prem-k-r

**Comment by prem-k-r** [comment]
Yep, but it's up to him.. if he likes or not

**Comment by itz-rj-here** [comment]
> Yep, but it's up to him.. if he likes or not

Im sure he would like your changes.

**Comment by XengShi** [comment]
> @giwih conflict 
> XengShi has modified the existing dark theme instead.
> It might be better to close this PR and open a new one focusing solely on the Russian translation.
> 
> 

Do you guys liked the new dark theme ?
@giwih @prem-k-r @itz-rj-here ?

**Comment by prem-k-r** [comment]
> > @giwih conflict
> > XengShi has modified the existing dark theme instead.
> > It might be better to close this PR and open a new one focusing solely on the Russian translation.
> 
> Do you guys liked the new dark theme ? @giwih @prem-k-r @itz-rj-here ?

Yep, almost perfect, it’s better than what we were suggesting.
But I think having two dark themes would’ve been great: one in light grey and the other in the black you changed it to. The grey theme needed some modifications too though.

**Comment by itz-rj-here** [comment]
> > @giwih conflict 
> > XengShi has modified the existing dark theme instead.
> > It might be better to close this PR and open a new one focusing solely on the Russian translation.
> > 
> > 
> 
> Do you guys liked the new dark theme ?
> @giwih @prem-k-r @itz-rj-here ?

Actually yeah looks better than the gray one. Using the gray one at night, it kind of hurts my eyes. Thats way i liked the black theme idea.

**Review by itz-rj-here — CHANGES_REQUESTED** [review]
Looks like the file has some conflicts :)

**Comment by prem-k-r** [comment]
> @giwih conflict 
XengShi has modified the existing dark theme instead. It might be better to close this PR and open a new one focusing solely on the Russian translation.

..

**Pull request closed without merging**